### PR TITLE
rm-guides

### PR DESCRIPTION
**Pull request by mango-db**
Removes unused guides content. Further changes expected by @rayangler in the guides revamp 🚀 

> "I wanna be that _guides._ / G. U. I." - Lady Gaga

### Staging Links:

[Realm](https://docs-mongodborg-staging.corp.mongodb.com/master/realm/cgoecknerwald/rm-guides/) for sanity

### Notes:
- There are lingering references to `guides` in some test data - they seem harmless
- The world wasn't ready for _Artpop_

### CODE REVIEW

File: src/utils/parse-marian-manifests.js [hunk 1]
```diff
@@ -11,7 +11,6 @@ const PROPERTY_MAPPING = {
   'docs-ruby': 'Ruby Driver',
   drivers: 'Drivers',
   ecosystem: 'Ecosystem',
-  guides: 'Guides',
```
Comment (rayangler) [inline src/utils/parse-marian-manifests.js:14]: Do we intentionally want to remove guides from being a part of search results?

Comment (mango-db) [reply src/utils/parse-marian-manifests.js:14]: I'm not sure - will leave in to be safe

**Comment by mango-db** [comment]
Staging has been updated to reflect changes

**Comment by mango-db** [comment]
Staging updated.

**Review by rayangler — APPROVED** [review]
LGTM! Thanks for the staging link for sanity

**Pull request merged**